sharing_portal/templates/sharing_portal/edit.html

```diff
@@ -35,7 +35,17 @@ <h2 class="artifactTitle">Edit artifact</h2>
         <h4>Authors</h4>
         {{ authors_formset.management_form }}
         {% for form in authors_formset.forms %}
-        <fieldset class="form-group form-inline">
+        <fieldset class="form-group form-inline" prefix="author">
+          {% bootstrap_form form show_label=False layout='inline' %}
+        </fieldset>
+        {% endfor %}
+        <h4>Roles</h4>
+        Assign roles to various users, allowing them to collaborate on this artifact. <br><br>
+        The <b>collaborator</b> role allows users to edit artifact metadata, upload new versions, and share private artifacts. <br><br>
+        The <b>administrator</b> role allows users to control almost everything about the artifact.<br><br>
```

Comment:
maybe we should say what exactly this means? It's unclear to me what "almost everything" refers to. Maybe we should link to a readthedocs paragraph explaining what each role can do.

Reply:
Agree it's not clear. I went with this phrasing because I couldn't think of a nice way to say "full control, except you can't get rid of the owner's admin role."

Reply:
Maybe what is most clear would be to have a table for these capabilities in the docs we can link to, and update there if things change.

Reply:
Good idea, done.